cetmix_tower_server/tests/test_variable.py

```diff
@@ -937,3 +938,157 @@ def test_validate_value(self):
             f"{variable_default_message.DEFAULT_VALIDATION_MESSAGE}",
             "Default validation message doesn't match",
         )
+
+
+class TestVariableReferenceRename(TestTowerCommon):
+    """Ensure {{ reference }} is updated everywhere after rename."""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.variable = cls.env["cx.tower.variable"].create(
```

Comment:
You are inheriting from the TowerCommon class, and it has [special variables](https://github.com/cetmix/cetmix-tower/blob/58529fe111deaf9a5012a1d7fab61f84c0a49fab/cetmix_tower_server/tests/common.py#L108) for that.